src/p2p/net_node.inl

```diff
@@ -1425,6 +1434,17 @@ namespace nodetool
     zone.m_notifier.on_handshake_complete(con->m_connection_id, con->m_is_income);
     zone.m_notifier.new_out_connection();
 
+    if (na.get_zone() == epee::net_utils::zone::tor) {
+      if (one_conn_below_max)
+      {
+        extend_make_next_connection_after(na.get_zone(), false);
```

Comment:
Also trivial, but you don't need the `if` branching here. Just `extend_make_next_connection_after(na.get_zone(), !one_con_below_max)`.

Reply:
Will change.